src/core/cita_bft.rs

```diff
@@ -256,7 +255,7 @@ impl TenderMint {
                 votes.push(vote);
             }
 
-            proto_proposal.set_lock_votes(RepeatedField::from_slice(&votes[..]));
+            proto_proposal.set_lock_votes(votes.into());
```

Comment:
Better use `Target::from`

Reply:
We want to remove protobuf dependence, the type is on protobuf ...